app/serializers/topic_view_serializer.rb

```diff
@@ -41,7 +41,8 @@ def self.attributes_from_topic(*list)
     :pinned_at,
     :pinned_until,
     :image_url,
-    :slow_mode_seconds
+    :slow_mode_seconds,
+    :external_id
```

Comment:
We might want an `include_external_id?` method to avoid adding it to JSON payloads if it's blank.

Reply:
Ya, that would probably be best.